cmd/describe.go

```diff
@@ -209,6 +260,14 @@ var describeIntersectionCmd = &cobra.Command{
 			return err
 		}
 
+		if dependents {
+			mods, err = mods.ExpandRequiredByDependencies()
```

Comment:
@buddyspike sorry for my many questions 😅 but would be right to wrap `mods` here in a manifest to apply filter on it, or should it be placed in a `ManifestByIntersection` method to be more inline with the other method ?

Reply:
No worries at all. I know, this is a confusing one. `intersection` is an interesting operation, I once thought is useful. But we didn't using it in our builds as we planned. We could skip filter capability in this command altogether for now if you are ok with it (I don't think anyone is using it).